Adyen.EcommLibrary/Model/Modification/CancelOrRefundRequest.cs

```diff
@@ -2,11 +2,22 @@
 using System.Collections.Generic;
 using System.Text;
 using Newtonsoft.Json;
+using Adyen.EcommLibrary.Constants;
 
 namespace Adyen.EcommLibrary.Model.Modification
 {
     public class CancelOrRefundRequest : AbstractModificationRequest
     {
+        public CancelOrRefundRequest()
+        {
+            var commonField = new CommonField
+            {
+                Name = ClientConfig.LibName, Version = ClientConfig.LibVersion
+            };
+            ApplicationInfo = new ApplicationInfo(commonField);
+        }

```

Comment:
as this is duplicate code in every request can we make it a method and put it into all payment service requests ?

Reply:
I moved the Application info to AbstractPaymentRequest and AbstractModificationRequest and removed it from every class request